glide.lock

```diff
@@ -2213,7 +2312,7 @@ imports:
   - pkg/apis/deviceplugin/v1beta1
   - pkg/apis/pluginregistration/v1
 - name: k8s.io/kubernetes
-  version: 5227b6f968eb46decfbb2d42e9827448a9b3910d
+  version: 3ee09fda8c72ef2489803eb69faec1e1f497302d
```

Comment:
you can't do that except for rebases

Reply:
To be more precise the vendor changes should be empty if publishing bot works. I am not sure if the bot also re-bumps origin glide to published levels, so this isolated change (without the vendor/k8s.io changes) might be fine.